elements/style.html

```diff
@@ -39,7 +39,7 @@
   <pre class="code-example">
     <code>
       &lt;head&gt;
-        &lt;h1&gt;<strong>&lt;style&gt;</strong>This content is styled in CSS<strong>&lt;style&gt;</strong>&lt;h1&gt;
+        &lt;h1&gt;<strong>&lt;style&gt;</strong>This content is styled in CSS<strong>&lt;/style&gt;</strong>&lt;/h1&gt;
```

Comment:
The example used for `<style>` is strange. I've seen `<p style=...` and `<style type=....` but I have not seen style used in the way shown.